velox/docs/functions/spark/datetime.rst

```diff
@@ -52,6 +52,13 @@ These functions support TIMESTAMP and DATE input types.
     Returns null if ``string`` does not match ``format`` or if ``format``
     is invalid.
 
+.. function:: week(x) -> integer
```

Comment:
The PR title says "week_of_year", but this doc says "week". Which is correct?

Reply:
fixed to `week_of_year`